dev/doc/nix-old-coq.md

```diff
@@ -0,0 +1,77 @@
+# Historic Coq versions using Nix
+
+## Introduction
+
+You can download and run older versions of Coq using Nix.
+
+Using Nix flakes, you can run coqide like:
+
+```sh
+nix shell nixpkgs#coqPackages_8_15.coqide -c coqide
+```
+
+This will setup and run CoqIDE 8.15.2 given a recent enough nixpkgs.
+
+Before 8.14, CoqIDE was included in the main Coq package so you only need to do:
+
+```sh
+nix shell nixpkgs#coq_8_12 -c coqide
+```
+
+TODO: nix-shell command should probably be here as well since most people will
```

Comment:
The equivalent `nix-shell` command is `nix-shell -p coq_8_15 coqPackages_8_15.coqide --run coqide`.